_uw-research-computing/htc-job-file-transfer.md

```diff
@@ -0,0 +1,101 @@
+
+> If you need a `/staging` directory, [request one here](quota-request).
+
+
+## Transferring Data to Jobs with `transfer_input_files`
+
+In the HTCondor submit file, `transfer_input_files` should always be used to tell HTCondor what files to transfer to each job, regardless of if that file originates from your `/home` or `/staging` directory. However, the syntax you use to tell HTCondor to fetch files from `/home` and `/staging` and transfer to your job will change depending on the file size.
+
+| Input Sizes | File Location |  Submit File Syntax to Transfer to Jobs |
+| ----------- | ----------- | ----------- | ----------- |
+| 0 - 100 MB      | `/home`       | `transfer_input_files = input.txt`       |
+| 100 MB - 30 GB   | `/staging`        | `transfer_input_files = osdf://chtc/staging/NetID/input.txt`        | 
+| > 30 GB   | `/staging`        | `transfer_input_files = file:///staging/NetID/input.txt`        | 
+| > 100 GB | | For larger datasets (100GB+ per job), contact the facilitation team about the best strategy to stage your data |
+
+Multiple input files and file transfer protocols can be specified and delimited by commas, as shown below:
+
+```
+# My job submit file
+
+transfer_input_files = file1, osdf://chtc/staging/username/file2, file:///staging/username/file3, dir1, dir2/
+
+... other submit file details ...
+```
+{:.sub}
+
+Ensure you are using the correct file transfer protocol for efficiency. Failure to use the right protocol can result in slow file transfers or overloading the system.
+
+### Important Note: File Transfers and Caching with `osdf://`
+The `osdf://` file transfer protocol uses a [caching](https://en.wikipedia.org/wiki/Cache_(computing)) mechanism for input files to reduce file transfers over the network. This can affect users who refer to input files that are frequently modified.
+
+*If you are changing the contents of the input files frequently, you should rename the file or change its path to ensure the new version is transferred.*
+
+## Transferring Data Back from Jobs to `/home` or `/staging`
+
+### Default Behavior for Transferring Output Files
+When a job completes, by default, HTCondor will return **newly created or edited files only in top-level directory** back to your `/home` directory. **Files in subdirectories are *not* transferred.** Ensure that the files you want are in the top-level directory by moving them or [creating tarballs](transfer-files-computer#c-transferring-multiple-files).
+
+### Specify Which Output Files to Transfer with `transfer_output_files` and `transfer_output_remaps`
+If you don't want to transfer all files but only *specific files*, in your HTCondor submit file, use
+```
+transfer_output_files = file1.txt, file2.txt
+```
+{:.sub}
+
```

Comment:
maybe add one example of how to use transfer_output_remaps for files in /home also?